src/three-components/SmoothControls.ts

```diff
@@ -42,6 +42,8 @@ export interface SmoothControlsOptions {
   eventHandlingBehavior?: EventHandlingBehavior;
   // Controls when interaction is allowed (always, or only when focused)
   interactionPolicy?: InteractionPolicy;
+  // The approximate size of the model (m)
+  modelSize?: number;
```

Comment:
Can you offer more context for this new configuration parameter?

Reply:
Nevermind, that's not needed. I can just reuse maximumRadius.